src/frontend/screens/WebView/index.tsx

```diff
@@ -197,7 +197,7 @@ export default function WebView() {
         partition="persist:epicstore"
         src={startUrl}
         allowpopups={trueAsStr}
-        useragent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) Chrome/200.0"
+        useragent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) Chrome/111.0"
```

Comment:
Decided to use a version that exists instead of a future number to avoid any other issue. But in my tests this does not make any difference.